lms/static/styles/ui-playground.scss

```diff
@@ -0,0 +1,53 @@
+// CSS Entry point for the local pattern library for LMS.
+// These styles are used in addition to the `frontend_apps` stylesheet.
+
+// The `frontend_apps` CSS uses utility and component styles, but not pattern styles.
+// Include them here so that the patterns in the Patterns section of pages
+// are styled.
+@use '@hypothesis/frontend-shared/styles/patterns';
+// Include the CSS for the Library components themselves
+@use '@hypothesis/frontend-shared/styles/library';
+
+// Application CSS styles the containing main content block as a centered flex column.
+// This looks peculiar with the pattern-library pages, which vary in width,
+// so override it.
+.content {
+  display: block;
+}
+
+// The following styles are temporary until the Pattern Library has a CSS
+// reset and is updated for base styles
+.LibraryExample__content,
+.LMSLibrary__content {
+  font-size: 1rem;
+  line-height: 1.4;
+
+  p {
```

Comment:
Everything above here already existed (this file was moved to a new location). Below here we have some temporary quick styles for the local pattern library until the CSS reset approach is made consistent across our apps. i.e. apply some styles to some now-unstyled library elements.